### DISCUSSION TITLE
Community Page UI/UX concepts

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [LemmyNet/lemmy-ui#1956] [Ideas]
### list of changes:

- moved the community name to be next to the logo
- moved the join community button to be next to the name
- moved the sidebar further to the top
- removed the original posts and comments buttons
- added the menu links
- moved the number of subscribers of the community to be under the name of the community
- moved the create post button to be next to the active button
![New Lemmy UI 3 2](https://github.com/LemmyNet/lemmy-ui/assets/98482469/39ac5d15-4466-4fc0-99b6-18688444218e)

**Comment by izz-j** [comment]
This looks really good!
